### PR TITLE
Increase Responses classes coverage

### PR DESCRIPTION
**Pull request by AvitalFineRedis**

### CODE REVIEW

File: src/test/java/redis/clients/jedis/resps/RespsTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package redis.clients.jedis.resps;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.*;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import redis.clients.jedis.GeoCoordinate;
+
+public class RespsTest {
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/resps/RespsTest.java]: ResponsesToStringTest

File: src/test/java/redis/clients/jedis/resps/RespsTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        assertThat(toStringResult, CoreMatchers.containsString("key-name"));
+        assertThat(toStringResult, CoreMatchers.containsString("elem"));
+        assertThat(toStringResult, CoreMatchers.containsString("1"));
+
+        // test hashCode
+        assertEquals(elem.hashCode(), elem_copy.hashCode());
+    }
+
+    @Test
+    public void TupleTest() {
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/resps/RespsTest.java]: There should be an existing TupleTest class. If that class covers this method, delete this. If not, move this there.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/test/java/redis/clients/jedis/resps/TupleTest.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    Tuple t2 = new Tuple("bar", 2d);
+
+    assertEquals(-1, t1.compareTo(t2));
+    assertEquals(1, t2.compareTo(t1));
+    assertNotEquals(t1, t2); // directly calling Tuple.equals()
+    assertNotEquals(t2, t1); // directly calling Tuple.equals()
+  }
+
+  @Test
+  public void compareSameObject() {
```
Comment (AvitalFineRedis) [inline src/test/java/redis/clients/jedis/resps/TupleTest.java:61]: This test was added

File: src/test/java/redis/clients/jedis/resps/TupleTest.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    Tuple t1 = new Tuple("foo", 1d);
+    assertEquals(t1, t1); // directly calling Tuple.equals()
+  }
+
+  @Test
+  public void compareDifferentType() {
```
Comment (AvitalFineRedis) [inline src/test/java/redis/clients/jedis/resps/TupleTest.java:67]: This test was added

File: src/test/java/redis/clients/jedis/resps/TupleTest.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    Tuple t1 = new Tuple("foo", 1d);
+    assertNotEquals(t1, new Object()); // directly calling Tuple.equals()
+    assertNotEquals(t1, null); // directly calling Tuple.equals()
+  }
+
+  @Test
+  public void testToString() {
```
Comment (AvitalFineRedis) [inline src/test/java/redis/clients/jedis/resps/TupleTest.java:74]: This test was added

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**